Snippets/ASB/ASB_7/MultipleNamespaces.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace ASB_7
+{
+    using NServiceBus;
+    using NServiceBus.AzureServiceBus.Addressing;
+
+    class MultipleNamespaces
+    {
+        private readonly EndpointConfiguration endpointConfiguration;
+
+        public MultipleNamespaces(EndpointConfiguration endpointConfiguration)
+        {
+            this.endpointConfiguration = endpointConfiguration;
```

Comment:
snippets dont need to be proper code. just pass in an `EndpointConfiguration endpointConfiguration` to each method if u need an insatnce